app/initial_state.js

```diff
@@ -73,22 +73,6 @@ const state = {
             },
         },
         general: {
-            server: {
```

Comment:
this is fixing the issue during development complaining about the non-existent stores in redux-persist